POSHI-615 Add method to escape regex chars

CalumR23 opened the pull request:
https://liferay.atlassian.net/browse/POSHI-615

allen-nagtalon committed:
POSHI-615 Add extra steps to format regex for textMatches

victorg1991 committed:
LPS-192713 Create date and time fragment

victorg1991 committed:
LPS-192713 Add default input

victorg1991 committed:
LPS-192713 Adapt to use date time

victorg1991 committed:
LPS-192713 Allow creating date and input fragments

victorg1991 committed:
LPS-192713 Create info field type

victorg1991 committed:
LPS-192713 Add FF

Mylena-Rodrigues committed:
LPS-192721 Do not display expired subscriptions in generate key order and sort list by available keys

shuyangzhou committed:
LPS-193018 Introduce a new sync point for initial request serving

shuyangzhou committed:
LPS-193018 Semantic versioning

shuyangzhou committed:
Revert "LPS-192359 An ugly workaround for the bad design. BundleSiteInitializer._addLayoutContent() is assuming the LayoutPageTemplateStructure is ready to use. But the LayoutPageTemplateStructure is created by previous Layout creation triggered by com.liferay.layout.page.template.internal.model.listener.LayoutModelListener.onAfterCreate(). During startup time with an empty db, BundleSiteInitializer may run before the LayoutModelListener is registered. Then the layout will be created without creating its LayoutPageTemplateStructure, causing BundleSiteInitializer._addLayoutContent() throws a NPE. By design, stuff created by ModelListener must be considered "optional", since other code simply has no knowledge about whether certain ModelListener exists or not. Any assumption about a ModelListener has previous invoked for an entity is fundamentally wrong."

This reverts commit 50b8c3fcf26f76fd66c2704c83cd929b583c153c.

shuyangzhou committed:
LPS-193018 Defer default layout creation to runtime to avoid startup time service availability racing. This also fixes various "No theme found for specified theme id XYZ. Returning the default theme." issues, like those reported in LPS-189121, which was caused by default layout creation racing with theme deployment.

shuyangzhou committed:
LPS-193018 Since the logic is defered to runtime, no need of these unused references anymore/

shuyangzhou committed:
LPS-193018 This was racing with BundleSiteInitializer's LayoutSet's lookAndFeel update during startup. Now BundleSiteInitializer is defered to runtime, this can run freely with no delay.

victorg1991 committed:
LPS-192713 Add tests case

Aline160 committed:
LPS-192803 Add logic to the card quantity count

shuyangzhou committed:
LPS-193018 Use bundle state to manage per company creation to take care misfire case that portal startup and shutdown without hitting InitFilter.

Commit:
LPS-192327 Improvements to Request entries

Commit:
LPS-163126 portal-search-web: Fix bug that search bar was not loaded in the home page

Basically when refactored, the developer missed the propertie isVisible to set
renderNothing so in pages that were not search templates, search bar was not
rendered when destination page dont exists

https://liferay.atlassian.net/browse/LPS-163126

CalumR23 committed:
POSHI-615 Add new method to escape regex characters

CalumR23 committed:
POSHI-615 Move method into RegexUtil

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

shuyangzhou committed:
LPS-193018 Abstract to base class for future reuse.

# Conflicts:
#	modules/apps/site/site-welcome-site-initializer/src/main/java/com/liferay/site/welcome/site/initializer/internal/instance/lifecycle/DefaultLayoutCreator.java

victoroscuro committed:
COMMERCE-12112 Fix assert on time

veroglez committed:
LPS-192410 Add focus-within to show the button when it is focused

victorg1991 committed:
LPS-192713 baseline

Kvale1733 committed:
LRQA-81647 Also exclude test from sqlserver release batch

shuyangzhou committed:
LPS-193018 Semantic versioning

brianchandotcom committed:
LPS-193018 Wordsmith

brianchandotcom committed:
Revert "LPS-189331 search-experiences-test: Add blank line to show that the order matters"

This reverts commit a5f223bd675b837fe7802566d630b7a0547e0c32.

brianchandotcom committed:
LPS-192713 SF

CalumR23 committed:
POSHI-615 Update com.liferay.poshi.core to 1.0.135

CalumR23 committed:
POSHI-615 FAKE GRADLE CACHE
